src/audits/performance/first-meaningful-paint.js

```diff
@@ -61,13 +67,13 @@ class FirstMeaningfulPaint extends Audit {
           // The fundamental Time To fMP metric
           const firstMeaningfulPaint = fmp.firstMeaningfulPaint - fmp.navigationStart;
 
-          // Roughly an exponential curve.
-          //   < 1000ms: penalty=0
-          //   3000ms: penalty=90
-          //   >= 5000ms: penalty=100
-          const power = (firstMeaningfulPaint - 1000) * 0.001 * 0.5;
-          const penalty = power > 0 ? Math.pow(10, power) : 0;
-          let score = 100 - penalty;
+          // Use the CDF of a log-normal distribution for scoring.
+          //   < 1100ms: score≈100
+          //   4000ms: score=50
+          //   >= 14000ms: score≈0
+          const distribution =
+              TracingProcessor.getLogNormalDistribution(SCORING_MEDIAN, SCORING_FALLOFF);
+          let score = 100 * distribution.computeComplementaryPercentile(firstMeaningfulPaint);
 
           // Clamp the score to 0 <= x <= 100.
           score = Math.min(100, score);
```

Comment:
it's possible without this that numerical error could creep in and we'd get the occasional -1 or 101 score, so it's probably better to leave it. OTOH, we *should* be fine because we `Math.round` below and the statistics package is careful with precision (there is no internal clamping to `[0,1]`, though)